documentation/Installing-Clear-Containers-on-Fedora-25.md

```diff
@@ -0,0 +1,60 @@
+#Install the Clear Containers runtime on Fedora 25
+
+##Install the Clear Containers runtime
+```
+$ sudo -E dnf config-manager --add-repo
+http://download.opensuse.org/repositories/home:clearlinux:preview:clear-containers-2.1/Fedora\_25/home:clearlinux:preview:clear-containers-2.1.repo
+$ sudo dnf install cc-oci-runtime linux-container
+```
+
+##Install docker 1.12
+```
+$ sudo -E dnf config-manager --add-repo https://yum.dockerproject.org/repo/testing/fedora/25
+$ sudo rpm --import https://yum.dockerproject.org/gpg
+$ sudo dnf install docker-engine-1.12.1-1.fc25
+```
+
+##Configure docker to use Clear Containers by default
+```
+$ sudo mkdir -p /etc/systemd/system/docker.service.d/
+$ sudo vi /etc/systemd/system/docker.service.d/clr-containers.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -D --add-runtime cor=/usr/bin/cc-oci-runtime --default-runtime=cor
+```
+##Restart the docker systemd service
+```
+$ sudo systemctl daemon-reload
+$ sudo systemctl restart docker
+```
+
+##Run a Clear Container
+You are now ready to run Clear Containers. For example
+```
+$ sudo docker run -ti fedora bash
+```
+##Notes
+You can override the default runtime docker will use by specifying
+`--runtime $runtime` (`$runtime` can be either `cor` or `runc` (dockers
+default)). For example, to specify explicitly that you wish to use the
+Clear Containers runtime
+```
+$ sudo docker run --runtime cor -ti ubuntu bash
+```
+##Increase limits (optional)
+If you are running a recent enough kernel (4.3+), you should consider
+increasing the `TasksMax=` systemd setting. Without this, the number of
+Clear Containers you are able to run will be limited.
```

Comment:
It might be a good idea to reference #154 here since the error you get if you *don't* specify `TasksMax` is not very helpful.